### ISSUE TITLE
Overfitting on Custom Dataset

### ISSUE THREAD

**Issue by hktxt** [ultralytics/yolov5#183]
I use this repo to train my own dataset, single class.
I got this result:
![f0_results](https://user-images.githubusercontent.com/16555750/85413356-4deb9880-b59d-11ea-9675-e75423600d05.png)
is it logical?

**Comment by yxNONG** [comment]
@hktxt is the class of your data include in coco? seem like it's over fitting, maybe just need few epoch and lower learning rate

**Comment by glenn-jocher** [comment]
@hktxt you get 0.94 mAP@0.5. You can use best.pt and call it a day, or you can experiment alternatives like @yxNONG mentions.
